README.md

```diff
@@ -1,163 +1,25 @@
+Welcome to **AWS Ambit Scenario Designer for Unreal Engine 4 (Ambit)**, a suite of tools to streamline 3D content creation at scale for autonomous vehicle and robotics simulation applications. From procedural placement of obstacles to automatic generation of whole cities, Ambit can help you quickly create the 3D environments and scenarios you need for your simulations or other 3D applications.
 
-## Directory Info
+![Ambit sample image](docs/readme-images/AmbitBanner.jpg)
 
-```
-📂 Ambit/                      ~ Source code for the Ambit plug-in
-📂 docs/                       ~ Source for Ambit documentation
-📂 HoudiniDigitalAssets/       ~ Source for Houdini digital assets
-📄 user-guide-buildspec.yml    ~ AWS CodeBuild configuration used to build the User Guide
-```
-
-## Developer Pre-Requisites
-
-- Unreal Engine 4.27
-- Visual Studio 2019
-- Houdini 18.5.696 (or higher)
-
-## Installation Instructions
-
-To use the Ambit plugin, place this repository in one of the two locations mentioned below:
-
-- In your Unreal Engine installation directory, under the `Engine > Plugins > Editor` folder. Choosing this method ensures the Ambit plugin is included in every Unreal Engine project you create. _Note: This is the less preferred option unless you also intend to build the engine from source._
-- If you want to include this plugin only for specific projects, add it to the `Plugins > Editor` folder for that specific Unreal project. These folders will need to be created if it doesn't already exist.
-
-## Dependencies
-
-### AWS SDK Module
-
-This repository contains the AWS SDK module called AWSUE4Module and pre-compiled aws-sdk-s3 libraries from [AWS SDK for C++ Version 1.9](https://github.com/aws/aws-sdk-cpp/tree/1.9.93).
-
-### Houdini Engine
-
-#### Installation
-
-In order to use the Houdini integration aspects of this code, you will need to download and install the Houdini Plugin (Version 2) for Unreal Engine. This plugin is bundled together with the Houdini installer on Windows. Follow these steps to get the plugin working inside Unreal Engine:
-
-1. Download and launch the Houdini installer. During the installation process, make sure the **Houdini Engine for Unreal** option is selected.
-2. Copy the **HoudiniEngine** folder to the `Plugins > Runtime` folder of either your Unreal directory, or your Unreal project directory.
-3. Launch Unreal.
+Review the [User Guide](https://aws-samples.github.io/aws-ambit-scenario-designer-ue4/) to learn about Ambit's capabilities and how to start using Ambit with your projects.
 
-For example, for UE 4.27 and Houdini 18.5.696, you will need to copy:
+If you're interested in contributing to Ambit's development or would like to customize Ambit's functionality to suite your team's needs be sure to read the ["Appendix: Contributing to Ambit Development"](https://aws-samples.github.io/appendix-contributing/) section of the User Guide.
```

Comment:
Link `https://aws-samples.github.io/appendix-contributing/` looks invalid. Please ensure that is the correct one.